rails/config/routes.rb

```diff
@@ -135,7 +164,13 @@
 
       resources :subjects
 
-      resources :teachers
+      resources :teachers do
+        member do
+          get :teacher_admin_or_manager
+          get :successful_creation
+          get :failed_creation
```

Comment:
I don't think any of these should be exposed. The first one is a before_filter.
The last two are methods called by `create` depending on success or failure.

Reply:
Yep - last commit trying to get this out before the weekend :)  I've removed them.